src/store/participantsStore.js

```diff
@@ -155,6 +155,14 @@ const getters = {
 
 		return {}
 	},
+
+	participantsInCall: (state) => (token) => {
+		if (state.inCall[token]) {
```

Comment:
As mentioned in 1-1, this is a temporary store to overwrite the "in call" flag for the current participant.
Do an array filter on `state.attendees[token]` with `attendee.inCall !== 0` and then count the length as currently.